docs/classification/pigeon-annotation.md

```diff
@@ -118,3 +118,50 @@ SIRV1	LexogenSIRVData	exon	7553	7814	.	-	0	gene_name "SIRV1"; gene_id "SIRV1"; t
 SIRV1	LexogenSIRVData	exon	10283	10366	.	-	0	gene_name "SIRV1"; gene_id "SIRV1"; transcript_id "SIRV103"; exon_assignment "SIRV103_4";
 SIRV1	LexogenSIRVData	exon	10648	10791	.	-	0	gene_name "SIRV1"; gene_id "SIRV1"; transcript_id "SIRV103"; exon_assignment "SIRV103_5";
 ```
+
+## Modification example
+
+Modifications can be made using a variety of commmand line tools, python, or gff specific tools such as [gffread](https://ccb.jhu.edu/software/stringtie/gff.shtml#gffread).
+
+Example of unmodified sorghum gff:
+```
+Chr01	cshl_gene	mRNA	2600	3653	.	+	.	ID=SbiRTX436.01G000100.1;Parent=SbiRTX436.01G000100;Name=SbiRTX436.01G000100.1;biotype=protein_coding
+Chr01	cshl_gene	five_prime_UTR	2600	2920	.	+	.	Parent=SbiRTX436.01G000100.1;Name=5UTR.1
+Chr01	cshl_gene	exon	2600	3653	.	+	.	Parent=SbiRTX436.01G000100.1;Name=exon.1
+Chr01	cshl_gene	CDS	2921	3586	.	+	.	Parent=SbiRTX436.01G000100.1;Name=CDS.1
+Chr01	cshl_gene	three_prime_UTR	3587	3653	.	+	.	Parent=SbiRTX436.01G000100.1;Name=3UTR.1
+```
+
+Update fields to adhere to above guidelines using python.
+```
+cat sorghum.gff | python -c '
+import sys
+for line in sys.stdin:
+  fields = line.split("\t")
+  if len(fields) > 8:
+    if fields[2] == "gene":
```

Comment:
Can you add a "gene" entry to the example, to show the "Name=" label changing? It doesn't show that at the moment.